server/views/pages/inPrisonCoursesAndQualifications/_completedCoursesTable.njk

```diff
@@ -0,0 +1,25 @@
+<table class="govuk-table" data-module="moj-sortable-table" data-qa="completed-courses-sortable-table">
```

Comment:
I've refactored the macro (now deleted 👇 ) into 3 simple nunjucks includes files.
It felt simpler this way, especially as there are subtle differences between each tab (completed, in progress, and withdrawn) when rendering the same field (start date; and initial sort order)